packages/slice-machine/lib/builders/common/Zone/Card/components/Hints/Renderers/vue.tsx

```diff
@@ -26,24 +26,24 @@ const codeByWidgetType = (
   [Widgets.LinkToMedia?.CUSTOM_NAME]: createPrismicLink,
   [Widgets.UID?.TYPE_NAME]: (fieldText: string) =>
     `<span>{{ ${fieldText} }}</span>`,
-  [Widgets.StructuredText?.TYPE_NAME]: createCodeFromTag("prismic-rich-text"),
-  [Widgets.Image?.TYPE_NAME]: createCodeFromTag("prismic-image"),
+  [Widgets.StructuredText?.TYPE_NAME]: createCodeFromTag("PrismicRichText"),
+  [Widgets.Image?.TYPE_NAME]: createCodeFromTag("PrismicImage"),
   [Widgets.Link?.TYPE_NAME]: createPrismicLink,
   [Widgets.Select?.TYPE_NAME]: createDefaultField(),
   [Widgets.Boolean?.TYPE_NAME]: (fieldText: string) =>
     `<span>{{ ${fieldText} ? 'true' : 'false' }}</span>`,
   [Widgets.Date?.TYPE_NAME]: (fieldText: string) =>
     `<span>{{ ${fieldText} }}</span>`,
   [Widgets.Timestamp?.TYPE_NAME]: createDefaultField(),
-  [Widgets.Embed?.TYPE_NAME]: createCodeFromTag("prismic-embed"),
+  [Widgets.Embed?.TYPE_NAME]: createCodeFromTag("PrismicEmbed"),
   [Widgets.Number?.TYPE_NAME]: createDefaultField(),
   [Widgets.GeoPoint?.TYPE_NAME]: createDefaultField(),
   [Widgets.Color?.TYPE_NAME]: (fieldText: string) =>
     `<span :style="\`color: \${${fieldText}}\`">Some Text</span>`,
   [Widgets.Text?.TYPE_NAME]: createDefaultField(),
 });
 
-const toVue: React.FC<{
+const toPreviousVue: React.FC<{
```

Comment:
is this not the most recent version of vue?

Reply:
Fixed :)